andela/travel_tool_front
# #161889102 Disable delete button  for delete checklist Item

#### What does this PR do?
This PR disables the `disable button` when delete reason for a checklist Item is empty
#### Description of Task to be completed?
- Add conditionals to the `disabled attribute` of the `disable button`

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the`DB` and on the `UserRoles` table, set yourself as a `Travel Admin`
- Go to the browser, refresh the page, Navigate to the `Trip planner` page
- Add a checklist Item if no checklist Item is present
- Click `disable` for a checklist item
- A confirmation modal should pop up and the `disable` button should be disabled

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161889102](https://www.pivotaltracker.com/story/show/161889102)

#### Screenshots (if appropriate)

![ddd](https://user-images.githubusercontent.com/27569442/48423240-88097d00-e760-11e8-9d01-3193c2f660e9.gif)


#### Questions:
N/A
